net/src/udp/mod.rs

```diff
@@ -128,6 +120,22 @@ impl Udp {
         self.0.length = length.get();
         self
     }
+
+    fn compute_checksum_ipv4(&self, net: &Ipv4, payload: impl AsRef<[u8]>) -> UdpChecksum {
+        #[allow(clippy::expect_used)] // payload greater than 2^16 bytes should be excluded by DPDK
+        self.0
+            .calc_checksum_ipv4(&net.0, payload.as_ref())
+            .expect("unreasonable payload")
```

Comment:
Can we be more specific than "unreasonable payload".

Also, is there a way not to panic here?  Maybe create an issue to not panic in these cases and just drop the packet?